tomb/src/pipelines/reconstruct.rs

```diff
@@ -68,7 +57,9 @@ pub async fn pipeline(
                     symlink(origin, built_path)?;
                 } else {
                     // If the parent does not yet exist
-                    if let Some(parent) = built_path.parent() && !parent.exists() {
+                    if let Some(parent) = built_path.parent()
+                        && !parent.exists()
+                    {
```

Comment:
Same as before... This is a really odd pattern and probably needs to be broken up into an early return